src/java/org/apache/cassandra/index/sai/disk/v1/postings/PostingListRangeIterator.java

```diff
@@ -118,6 +118,7 @@ protected PrimaryKey computeNext()
             if (!(t instanceof QueryCancelledException))
                 logger.error(indexIdentifier.logMessage("Unable to provide next token!"), t);
 
+            FileUtils.closeQuietly(Arrays.asList(postingList, primaryKeyMap));
```

Comment:
Looks like we're just following the pattern of `LiteralIndexSegmentTermsReader` and `BlockBalancedTreeReader`. Makes sense.